packages/core/src/styles/_type.scss

```diff
@@ -0,0 +1,100 @@
+@import "~@carbon/themes/scss/themes";
+@import "~@carbon/type/scss/type";
+@import "~@carbon/type/scss/font-face/mono";
+@import "~@carbon/type/scss/font-face/sans";
+
+@include carbon--type-reset();
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+svg.chart-svg {
+	font-family: carbon--font-family('sans');
+
+	text {
+		font-size: 12px;
+		font-weight: carbon--font-weight('regular');
+
+		&.chart-label {
+			font-family: "IBM Plex Sans Condensed", "IBM Plex Sans", sans-serif;
+		}
+
+		&.title {
+			font-size: 18px;
+			font-weight: carbon--font-weight('semibold');
+		}
+	}
+
+	g.legend-item text {
+		font-size: 15px;
+	}
+
+	g.tick {
+		text {
+			font-family: carbon--font-family('sans-condensed');
+		}
+	}
+ }
+
+
+/////////////////////////////////////////////
+///          Theme styles                ////
+/////////////////////////////////////////////
+
+.carbon--theme--white {
+	text {
+		fill: $text-02;
+
+		&.axis-label,
+		&.chart-title {
+			fill: $text-01;
+			stroke: $text-01;
+		}
+	}
+}
+
+.carbon--theme--g10 {
+	@include carbon--theme($carbon--theme--g10) {
+		text {
+			fill: $text-02;
+
+			&.axis-label,
+			&.chart-title {
+				fill: $text-01;
+				stroke: $text-01;
+			}
+		}
+	}
+}
+
+.carbon--theme--g90 {
+	@include carbon--theme($carbon--theme--g90) {
+		text {
+			fill: $text-02;
+
+			&.axis-label,
```

Comment:
Are these scoped well enough to prevent style leakage?

Reply:
👍 good point, ill encompass it wihin the svg.chart-svg class